administrative-sdk/organisation.js

```diff
@@ -20,67 +20,24 @@ class Organisation {
   }
 
   /**
-   * Callback used by createOrganisation.
-   *
-   * @callback Sdk~organisationCreatedCallback
-   * @param {its.Organisation} organisation Updated organisation domain model instance.
-   */
-  organisationCreatedCallback(organisation) {}
-
-  /**
-   * Error callback used by createOrganisation.
-   *
-   * @callback Sdk~organisationCreatedErrorCallback
-   * @param {object[]} errors Array of errors.
-   * @param {its.Organisation} organisation Organisation domain model instance with unapplied changes.
-   */
-  organisationCreatedErrorCallback(errors, organisation) {}
-
-  /**
    * Create an organisation.
    *
    * @param {its.Organisation} organisation An organisation domain model instance.
    * @param {Sdk~organisationCreatedCallback} [cb] The callback that handles the response.
    * @param {Sdk~organisationCreatedErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.